pkgs/applications/version-management/git-and-tools/git-quick-stats/default.nix

```
@@ -0,0 +1,42 @@
+{ stdenv, fetchFromGitHub, makeWrapper, coreutils, git, gnugrep, gnused }:
+
+stdenv.mkDerivation rec {
+    pname = "git-quick-stats";
+    version = "2.0.8";
+
+    src = fetchFromGitHub {
+        owner = "arzzen";
+        repo = pname;
+        rev = "c11bce17bdb1c7f1272c556605b48770646bf807";
+        sha256 = "1px1sk7b6mjnbclsr1jn33m9k4wd8wqyw4d6w1rgj0ii29lhzmqi";
+    };
+
+    buildInputs = [ makeWrapper ];
+
+    dontBuild = true;
+
+    installPhase = ''
+        mkdir -p $out/bin
+        cp -a git-quick-stats $out/bin/git-quick-stats
+    '';
+
+    wrapperPath = with stdenv.lib; makeBinPath [
+        coreutils
+        git
+        gnugrep
+        gnused
+    ];
+
+    fixupPhase = ''
```

Overriding phases breaks the default hooks.

```suggestion
    postFixup = ''
```